pkg/metrics/features/metrics.go

```diff
@@ -547,4 +576,18 @@ func (m Metrics) update(params enabledFeatures, config *option.DaemonConfig) {
 	if config.EnableEnvoyConfig {
 		m.ACLBCiliumEnvoyConfigEnabled.Add(1)
 	}
+
+	var bigTCPProto string
+	switch {
+	case params.BigTCPConfig().IsIPv4Enabled() && params.BigTCPConfig().IsIPv6Enabled():
+		bigTCPProto = advConnBigTCPDualStack
+	case params.BigTCPConfig().IsIPv4Enabled():
+		bigTCPProto = advConnBigTCPIPv4
+	case params.BigTCPConfig().IsIPv6Enabled():
+		bigTCPProto = advConnBigTCPIPv6
+	}
```

Comment:
Do we need DualStack one, meaning, is it not enough to report that IPv4 + IPv6 is enabled each or can this only have one value and in this case we need DualStack?

Reply:
It can only have one value and that's why we need "DualStack".